openbb_terminal/forex/forex_controller.py

```diff
@@ -75,6 +75,7 @@ def __init__(self, queue: Optional[List[str]] = None):
         if session and get_current_user().preferences.USE_PROMPT_TOOLKIT:
             choices: dict = self.choices_default
             choices["load"].update({c: {} for c in FX_TICKERS})
+            del choices["load"]["--source"]["Oanda"]
```

Comment:
Since this is autogenerated, you should be able to take Oanda from the sources instead of deleting it here.

Reply:
Thanks for spotting this!